servers/publikator/lib/mailchimp/createCampaign.js

```diff
@@ -30,6 +21,10 @@ module.exports = async ({ title, subject }) => {
       }
     })
   })
-    .then(response => response.json())
-    .catch(error => console.error('updateMailchimp failed', { error }))
+    .then(response => {
+      if (!response.ok) {
+        throw Error(response.statusText)
+      }
+      return response
+    })
```

Comment:
There was [a check](https://github.com/orbiting/backends/pull/436/files#diff-6e358742c5bc46972ccb65ea7e02ad7fL277) for `status !== 'save'` before, would it not be worth keeping that to ensure mailchimp did really save not just the request didn't fail?

Reply:
MailChimp will either return a response which is thrown as an error if `response.ok` is falsy ([status code outside 200-299 range](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok)), or fetch itself will throw an error.

Thus sticking to current setup, believing it handles MailChimp API (and fetch) rather well.